ani-cli

```diff
@@ -464,7 +464,7 @@ scrape=Query
 debug=0
 choice=
 auto_play=0
-project_root="$(dirname "$0")"
+project_root="$(dirname "$0" | sed 's/....$//')/share/ani-cli"
```

Comment:
See my comment from earlier, this is not how this part of the code works

Reply:
im basically cloning the master branch to share/ani-cli and symlinking bin/ani-cli to share/ani-cli/ani-cli.
im currently using on macos, linux and android and everything seems to work.
heres the repo/branch im using: https://github.com/DaBigBlob/ani-cli/tree/fixed